### PR TITLE
Added CR links in APIs for MAUI SfPulltoRefresh

### PR DESCRIPTION
**Pull request by AnanthaLakshmiKannan**
Added CR links in APIs for MAUI SfPulltoRefresh

### CODE REVIEW

File: MAUI/Pull-to-Refresh/getting-started.md [hunk 1]
```diff
@@ -82,7 +82,7 @@ The `Syncfusion.Maui.Core` NuGet is a dependent package for all Syncfusion contr
 
 ## Defining PullableConent
 
-`SfPullToRefresh.PullableContent` view serves as the designated area where users can initiate the pull-to-refresh action, enabling them to refresh and update the content within the view.
+[SfPullToRefresh.PullableContent](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.PullToRefresh.SfPullToRefresh.html) view serves as the designated area where users can initiate the pull-to-refresh action, enabling them to refresh and update the content within the view.
```
Comment (PiruthivirajM) [inline MAUI/Pull-to-Refresh/getting-started.md:85]: link is mismatched.

File: MAUI/Pull-to-Refresh/getting-started.md [hunk 2]
```diff
@@ -15,8 +15,8 @@ This section provides a quick overview of how to get started with the .NET MAUI
 
  1. Create a new .NET MAUI application in Visual Studio.
  2. Syncfusion .NET MAUI components are available on [nuget.org](https://www.nuget.org/). To add SfPullToRefresh to your project, open the NuGet package manager in Visual Studio, search for Syncfusion.Maui.PullToRefresh and install it.
- 3. Import the control namespace `Syncfusion.Maui.PullToRefresh` in XAML or C# code.
- 4. Initialize the `SfPullToRefresh` control.
+ 3. Import the control namespace [Syncfusion.Maui.PullToRefresh](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.PullToRefresh.html) in XAML or C# code.
```
Comment (PiruthivirajM) [inline MAUI/Pull-to-Refresh/getting-started.md:18]: Remove both links here and add it above topic.

**Review by PiruthivirajM — CHANGES_REQUESTED** [review]
Resolve the comments.

File: MAUI/Pull-to-Refresh/getting-started.md (hunk 1 above)
Comment (AnanthaLakshmiKannan) [reply MAUI/Pull-to-Refresh/getting-started.md]: Correct link updated

File: MAUI/Pull-to-Refresh/getting-started.md (hunk 2 above)
Comment (AnanthaLakshmiKannan) [reply MAUI/Pull-to-Refresh/getting-started.md]: Removed links and updated in above topic

**Review by PiruthivirajM — APPROVED** [review]

**Pull request merged**